Firefly/Views/Transactions/TransactionCreate.swift

```diff
@@ -173,6 +175,7 @@ struct TransactionCreate: View {
     }
 
     func submitTransaction() async throws {
+        submitIsLoading = true
```

Comment:
**suggestion (bug_risk):** Reset loading state on error

Ensure that `submitIsLoading` is set to false in the catch block as well. This will prevent the UI from staying in a loading state if an error occurs during submission.